tests/kafkatest/tests/streams/streams_broker_bounce_test.py

```diff
@@ -284,8 +272,8 @@ def test_many_brokers_bounce(self, failure_mode, num_failures, metadata_quorum=q
     @cluster(num_nodes=7)
     @matrix(failure_mode=["clean_bounce", "hard_bounce"],
             num_failures=[3],
-            metadata_quorum=quorum.all_non_upgrade)
-    def test_all_brokers_bounce(self, failure_mode, num_failures, metadata_quorum=quorum.zk):
+            metadata_quorum=[quorum.isolated_kraft])
```

Comment:
as above

Reply:
Required to indicate to use KRaft for metadata quorum.  Failing to do so without a `@matrix` annotation results in this error as the test will default to ZK without it.
```File "/opt/kafka-dev/tests/kafkatest/services/kafka/kafka.py", line 541, in security_config
    zk_sasl=self.zk.zk_sasl if self.quorum_info.using_zk else False, zk_tls=self.zk_client_secure,
AttributeError: 'NoneType' object has no attribute 'zk_sasl'
```

Reply:
update: some of these tests [were failing ](https://semaphore.ci.confluent.io/jobs/f0ad9c45-d5fd-433b-a2d7-72d8e36440ff/summary?report_id=0356f43b-c807-318c-b61e-09e8fdd4e976) as previously configured so I needed to update to `isolated_kraft` and increase the number of nodes.

![many_brokers_bounce_failures_part_1](https://github.com/user-attachments/assets/835966be-a5fe-444c-9db7-4b9ec363c319)